package.json

```diff
@@ -6,7 +6,9 @@
   "jsnext:main": "esm/index.js",
   "typings": "index.d.ts",
   "dependencies": {
-    "@angular/core": ">= 2.0.0-rc.5"
+    "@angular/core": ">= 2.0.0",
+    "rxjs": "5.0.0-beta.12",
```

Comment:
this package is not depending on rxjs nor zone.js, \@angular/core is depending on them (and even as peers) so it will reference them and this package should not